client/java-client-osgi/pom.xml

```diff
@@ -164,6 +166,9 @@
               <artifact>*:*</artifact>
               <excludes>
                 <exclude>module-info.class</exclude>
+                <exclude>logback-shared.xml</exclude>
+                <exclude>plugins_min_versions.txt</exclude>
```

Comment:
The `plugins_min_versions.txt` file will be removed in this PR:
https://github.com/SonarSource/sonarlint-core/pull/1150
It will be merged soon.

Can you please wait until it is merged and remove this line from your PR?

Reply:
Sure, I will rebase then.

Reply:
Merged